### DISCUSSION TITLE
[Bug] 创建java网站的运行目录问题

### DISCUSSION THREAD

**Discussion by duxdyu** [1Panel-dev/1Panel#7156] [Q&A]
### 联系方式

350584671@qq.com

### 1Panel 版本

v1.10.21-lts 

### 问题描述

使用java环境搭建网站，在网站标签里，点击网站目录跳转，和在运行环境标签里点击网站目录跳转不是一个位置

### 重现步骤

1.在运行环境标签创建java运行环境，这时候必须选一个运行目录，这个目录要自己建，但是这个运行目录不能是/opt/1panel/apps/openresty/openresty/www/sites/域名/index，不然一会创建网站就创建不了
2.在网站标签创建网站，这时候点运行目录，就会跳到一个空文件夹
3.在运行环境的运行目录跳转，就可以跳转到正确的位置

### 期待的正确结果

不管是在网站标签里，还是在运行环境标签里，点击网站目录跳转应该是到一个位置

**Comment by zhengkunwang223** [comment]
其实网站目录原本的跳转目录是  /opt/1panel/apps/openresty/openresty/www/sites/域名 
这个目录下会有 index log proxy rewrite 等等目录 
后来为了方便大多数 php 网站用户，跳转到了 index 目录
而运行环境的目录 为 源码目录 或者是 应用包的运行目录 
两者概念不一样 所以不能跳到同一个路径
